scripts/st2bootstrap-deb.template.sh

```diff
@@ -155,6 +155,10 @@ install_st2_dependencies() {
 
   sudo apt-get install -y curl
   sudo apt-get install -y rabbitmq-server
+  sudo rabbitmqctl add_user stanley "${ST2_RABBITMQ_PASSWORD}"
+  sudo rabbitmqctl delete_user guest
+  rabbitmqctl set_user_tags stanley administrator
+  rabbitmqctl set_permissions -p / stanley ".*" ".*" ".*"
 
   # Configure RabbitMQ to listen on localhost only
   sudo sh -c 'echo "RABBITMQ_NODE_IP_ADDRESS=127.0.0.1" >> /etc/rabbitmq/rabbitmq-env.conf'
```

Comment:
As we add new RabbitMQ configuration steps I think at this point the logic grew enough to make it a dedicated bash function `install_rabbitmq()` which would include repository add, installation, and configuration.

See `install_mongodb()` for the example.